### ISSUE TITLE
how to set the l2penalty in the code?

### ISSUE THREAD

**Issue by 040840308** [carlini/audio_adversarial_examples#16]
Hi, 

        Does anyone know the setting of l2penalty? The default l2penalty is infinite and the following if branch (in attack.py) will never be executed:
       if not np.isinf(l2penalty):
                loss = tf.reduce_mean((self.new_input-self.original)**2,axis=1) + l2penalty*ctcloss
            else:
                loss = ctcloss

        Thanks for any advice.

**Comment by carlini** [comment]
You can pass --l2penalty and supply any other value you want.

https://github.com/carlini/audio_adversarial_examples/blob/master/attack.py#L320

**Comment by 040840308** [comment]
I know that I can pass any value to this argument. However, I don't have experience about this. What value do you often set this? 

Thanks

**Comment by carlini** [comment]
Start with it at np.inf. It works nearly as well. You can start to reduce it and see what value you can set it at so that it still works. Smaller is better here. I usually have it somewhere around 10-100.

**Comment by 040840308** [comment]
Nice, I will have a try. Yesterday, I set it to 10000. But, it does not work well.
Thanks

**Comment by carlini** [comment]
If it doesn't work at 10000 then at 100 won't work either, although it's surprising that such a large value doesn't work.

**Comment by 040840308** [comment]
Anyway, I will have a try according to your advice. Maybe, somethings is overlooked.
